static/src/js/pages/HomePage/HomePage.jsx

```diff
@@ -0,0 +1,119 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import Col from 'react-bootstrap/Col'
+import Row from 'react-bootstrap/Row'
+import { FaQuestionCircle, FaSignInAlt } from 'react-icons/fa'
+
+import Page from '../../components/Page/Page'
+
+import './HomePage.scss'
+import For from '../../components/For/For'
+import Button from '../../components/Button/Button'
+import useAppContext from '../../hooks/useAppContext'
+
+/**
+ * Renders a `HomePagePanel` component.
+ *
+ * This is intended be used within the `HomePage` component.
+ *
+ * @component
+ * @example <caption>Renders a `HomePagePanel` component</caption>
+ * return (
+ *   <HomePagePanel title="This will display as the title">
+ *     This text will be in the body of the panel
+ *   </HomePagePanel>
+ * )
+ */
+const HomePagePanel = ({
+  children,
+  title
+}) => (
+  <Col className="d-flex mb-4" sm={12} md={6} lg={5}>
+    <div
+      className="home-page__panel-content bg-black text-white p-4 border-start border-4 border-pink"
+      style={{ '--bs-text-opacity': 0.95 }}
+    >
+      <div className="d-flex align-items-center mb-3">
+        <FaQuestionCircle className="me-2 d-block text-opacity-75" />
+        <h2 className="h6 mb-0 text-uppercase fw-bold">
+          {' '}
+          {title}
+        </h2>
+      </div>
+      <p className="mb-0 small">{children}</p>
+    </div>
+  </Col>
+)
+
+HomePagePanel.propTypes = {
+  children: PropTypes.node.isRequired,
+  title: PropTypes.string.isRequired
+}
+
+/**
+ * Renders a `HomePage` component
+ *
+ * @component
+ * @example <caption>Renders a `HomePage` component</caption>
+ * return (
+ *   <HomePage />
+ * )
+ */
+const HomePage = () => {
```

Comment:
Should this be another component?

Reply:
Thought on this some more. This is in the `pages` directory, so having a component there might not be ideal. I'll break it out into a separate generic panel component. Thanks for the callout!